conode/conode.go

```diff
@@ -22,6 +22,7 @@ import (
 	_ "github.com/dedis/cothority/cosi/service"
 	_ "github.com/dedis/cothority/guard/service"
 	_ "github.com/dedis/cothority/identity"
+	_ "github.com/dedis/cothority/pop/service"
```

Comment:
Mhhhh... I'm not so sure about that. What about having a "demo" folder or something that clearly marks that at least this pop service has been hacked in 3 days  for a demo and is in no way more than a demo ? 
I'm not so sure about including it in the default list of services to offer when running a conode...